rootio/scanner.go

```diff
@@ -57,25 +57,25 @@ func (s *baseScanner) SeekEntry(i int64) error {
 	if s.chain {
 		if s.cur >= s.n {
 			itree := s.findTree(i)
-			s.loadTree(itree)
+			if itree != -1 {
```

Comment:
I would actually reverse the test. we do want to propagate back to the user that we couldn't find the entry she was desperately needing to get her Nobel prize.
```go
if itree == -1 {
    s.err = fmt.Errorf("rootio: could not find Tree for entry %d", i)
    return s.err
}
s.loadTree(itree)
```